tests/open/24.t

```diff
@@ -2,17 +2,31 @@
 # vim: filetype=sh noexpandtab ts=8 sw=8
 # $FreeBSD: head/tools/regression/pjdfstest/tests/open/24.t 211352 2010-08-15 21:24:17Z pjd $
 
-desc="open returns EOPNOTSUPP when trying to open UNIX domain socket"
-
 dir=`dirname $0`
 . ${dir}/../misc.sh
 
+# POSIX doesn't explicitly state the errno for open(2)'ing sockets.
+case ${os} in
+Darwin|FreeBSD)
+	expected_error=EOPNOTSUPP
+	;;
+Linux)
+	expected_error=ENXIO
+	;;
+*)
+	echo "1..0 # SKIP: unsupported OS: ${os}"
```

Comment:
`quick_exit` relies on an abnormality in the TAP syntax (specify a test plan of `1..1` and exit). This is pedantically incorrect (but not protected again) and can cause issues if multiple test plans are printed out or 

Using the `Bail out!` syntax is correct in many cases because it acknowledges that there might have been a test plan/results reported, and the test is being terminated abnormally.

In this case though, skip is right, because we technically haven't done anything yet.

Reference: https://testanything.org/tap-specification.html